pkg/server/commands/listusers/list_users_rpc_test.go

```diff
@@ -29,6 +29,7 @@ type ListUsersTests []struct {
 	tuples                []*openfgav1.TupleKey
 	expectedUsers         []string
 	expectedErrorMsg      string
+	context               map[string]interface{}
```

Comment:
Instead of passing this as a separate parameter, can we please pass it as part of the request it is associated with? We do this elsewhere with the testutil helper:

```go
req: &openfgav1.ListUsersRequest{
    ...,
    Context: testutils.MustNewStruct(t, map[string]interface{}{...})
}
```